cmd/swagger/commands/generate/contrib.go

```diff
@@ -0,0 +1,14 @@
+package generate
+
+import "github.com/go-swagger/go-swagger/generator"
+
+// contribOptionsOverride gives contributed templates the ability to override the options if they need
+func contribOptionsOverride(opts *generator.GenOpts) {
+	switch opts.Template {
+	case "stratoscale":
+		// Stratoscale template needs to regenerate the configureapi on every run.
+		opts.RegenerateConfigureAPI = true
```

Comment:
This may be overriden directly in the section running templates. There is a `SkipExists` flag on each template.

Reply:
You have to consider that the user might chose to override the stratoscale template with a derivation of her own making, override again this skip decision for some reason.

I would say that we should just override the defaults in the templates section before we load the user's own config file.

Reply:
OK, since this option is now available for every template to override the config setting, contrib template or not.